lib/auth/auth_with_roles.go

```diff
@@ -5142,6 +5251,13 @@ func (a *ServerWithRoles) GenerateUserSingleUseCerts(ctx context.Context) (proto
 	return nil, trace.NotImplemented("bug: GenerateUserSingleUseCerts must not be called on auth.ServerWithRoles")
 }
 
+// GetUnifiedResources exists to satisfy auth.ClientI but is not
+// implemented here. It is a client only interface to make
+// interacting with ListUnifiedResources friendlier.
+func (a *ServerWithRoles) GetUnifiedResources(ctx context.Context, req *proto.ListUnifiedResourcesRequest) (*proto.ListUnifiedResourcesResponse, error) {
```

Comment:
I'll let this one pass, since I see a similar pattern already in use around here, but I'd be glad to understand why we have these functions in the interface in the first place if we are not supposed to call them. This probably calls for a better interface segregation.